examples/hotrod/services/driver/server.go

```diff
@@ -72,31 +77,57 @@ func (s *Server) Run() error {
 	select {}
 }
 
-// FindNearest implements Thrift interface TChanDriver
+// FindNearest implements Thrift interface TChanDriver. config.GetDriversConcurrently adds concurrency.
 func (s *Server) FindNearest(ctx thrift.Context, location string) ([]*driver.DriverLocation, error) {
 	s.logger.For(ctx).Info("Searching for nearby drivers", zap.String("location", location))
 	driverIDs := s.redis.FindDriverIDs(ctx, location)
-
 	retMe := make([]*driver.DriverLocation, len(driverIDs))
-	for i, driverID := range driverIDs {
-		var drv Driver
+	var topErr error
```

Comment:
there seems to be a lot of unnecessary complexity here. See how `fixRouteWorkerPoolSize` is used, this should be exactly the same, using an executor pool.

Reply:
Got it. Will change. Thanks.